modules/ob-configuring-pods-in-build-run.adoc

```diff
@@ -0,0 +1,21 @@
+// This module is included in the following assembly:
+//
+// * configuring/configuring-build-runs.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ob-configuring-pods-in-build-run_{context}"]
+= Pod configuration
+
+You can use the following optional `BuildRun` CR fields to configure {builds-shortname} pods:
```

Comment:
In the preview, I see that `{builds-shortname} pods` renders as `Builds pods`. Is that correct, or should it be `Build pods`? I ask because in the assembly files in this PR, I see the phrase `build pods` used.

Reply:
it should be Builds pods because it refers to pods created with the product (aka Builds).